### PR TITLE
add some informations / examples in documentation

### PR DESCRIPTION
**Pull request by AnaelleBernard**

### CODE REVIEW

**Comment by AnaelleBernard** [comment]
multi examples errors, string act like a list
image

File: api.yaml [hunk 1]
```diff
@@ -89,7 +96,9 @@ components:
           description: Id in your CRM or SI of the request of the lead
         resume:
           type: string 
-          description: Pinpo importants informations
+          description: |
+           Pinpo importants informations <br/>
```
Comment (fbeiger) [inline api.yaml:100]: "Pinpo relevant information"

**Review by fbeiger — CHANGES_REQUESTED** [review]

**Review by Jean-CharlesTerrillon — APPROVED** [review]

File: api.yaml [hunk 2]
```diff
@@ -255,18 +264,17 @@ components:
           example: jiu7897987GUYGhh
           description: Id of saleperson in you CRM
         iCalUrl:
-           type: string
-           example: https://joiuezhfiuhIcal.fr
-           description: iCalUrl of salesperson assigned to lead can be used to check saleperson disponibilities
+          type: string
+          example: https://joiuezhfiuhIcal.fr
+          description: iCalUrl of salesperson assigned to lead can be used to check saleperson disponibilities
```
Comment (Jean-CharlesTerrillon) [inline api.yaml:269]:
```suggestion
          description: iCalUrl of salesperson assigned to lead can be used to check saleperson availabilities
```

**Review by Jean-CharlesTerrillon — CHANGES_REQUESTED** [review]

File: api.yaml (hunk 2 above)
Comment (AnaelleBernard) [reply api.yaml]: done

File: api.yaml (hunk 1 above)
Comment (AnaelleBernard) [reply api.yaml]: done

**Review by Jean-CharlesTerrillon — APPROVED** [review]

**Pull request merged**